### DISCUSSION TITLE
Checkpatch lint does not appear on Mason lint list.

### DISCUSSION THREAD

**Discussion by rodrigopex** [LazyVim/LazyVim#4610] [Q&A]
Hi all. Today, I tried to install the `checkpatch` lint using Mason, but it does not appear in my list. I double-checked, and I could see the lint configuration on the `nvim-lint` repo. I also double-checked that the' check patch is in the `share` folder populated by LazyVim. Does anybody have any idea what is happening? I want to use the IDE with almost no hacks because that is what I like on LazyVim. I appreciate any help you can provide.

**Comment by dpetka2001** [comment]
This linter is not available on Mason, so there's nothing you can do about that. You should just install the linter via other means and then configure in `nvim-lint` according to what it says at the file `checkpatch.lua`. Or make a feature request in Mason to make available `checkpatch` as a package.

**Reply by rodrigopex** [reply]
The funny thing is that the linter seems available on the `nvim-lint` like the others, but it is not there. Thanks for answering. I will try something here. Regards.
